helmfile/values/grafana-user.yaml.gotmpl

```diff
@@ -70,6 +71,22 @@ datasources:
       type: prometheus
       url: http://kube-prometheus-stack-prometheus:9090
       version: 1
+    {{- end }}
+    {{- if and .Values.thanos.enabled .Values.thanos.receiver.enabled }}
+    {{- range .Values.global.clustersMonitoring }}
+    - name: "Ck8s metrics"
```

Comment:
we may also need sc data source, e.g.: elasticsearch dashboard needs sc metrics

Reply:
Right, they'll need it for that. I thought we could get away from them needing sc cluster data. But I'll add that for them